src/components/inputs/Search/SearchBar/SearchBar.tsx

```diff
@@ -84,6 +104,19 @@ function SearchBar(props: SearchBarProps) {
       </div>
       {React.cloneElement(footerElement, {
         ...props,
+        defaultText: (
+          <>
+            {!isValid ? (
```

Comment:
let's pass this to the `SearchBarFooter` element and put this div in there

Reply:
added